# variables of the message

string last_event
string event_type

uint16_t[] param 

string[] possible_events
